src/content/en/updates/2019/02/back-forward-cache.md

```diff
@@ -0,0 +1,44 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: bfcache creates a cache allowing for instant navigations to previously-visited pages.
+
+{# wf_updated_on: 2019-02-14 #}
+{# wf_published_on: 2019-02-15 #}
+{# wf_tags: performance #}
+{# wf_featured_image: /web/updates/images/generic/devices.png #}
+{# wf_featured_snippet: On the Chrome team, we are exploring a new back/forward cache to cache pages in-memory (preserving JavaScript & DOM state) when the user navigates away. #}
+{# wf_blink_components: Blink #}
+
+# Exploring a back/forward cache for Chrome {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+On the Chrome team, we are exploring a new **[back/forward cache](https://www.chromestatus.com/feature/5815270035685376)** to cache pages in-memory (preserving JavaScript & DOM state) when the user navigates away. This is definitely not a trivial endeavor but if it succeeds it will make **navigating back and forth** *very* fast. 
+
+A back/forward cache (bfcache) caches whole pages (including the JavaScript heap) when navigating away from a page, so that the full state of the page can be restored when the user navigates back. Think of it as **pausing** a page when you leave it and **playing** it when you return.
+
+Below is a first-look of an **early prototype** of back/forward cache in action on desktop:
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="eusKoHPAWtU"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+We also have a preview of the back/forward cache working on Chrome for Android:
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="cuPsdRckkF0"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+We estimate this change could **improve performance for 19% of all navigations** for mobile Chrome. You can find more detail about this feature in the [bfcache explainer](https://docs.google.com/document/d/1mrgp7XzR16rd1xqFYOJgC1IP0NPLZFaRU5Ukj3-TlLw/edit#heading=h.d9wqdzopmdcf).
```

Comment:
Perhaps, "up to 19%" given that we expect to go at it in steps up until we hit diminishing returns (e.g. corner cases, bailing out cases).

Reply:
Thanks, Kenji! Updating now.